docs/excel/custom-functions-dialog.md

```diff
@@ -0,0 +1,105 @@
+---
+ms.date: 03/13/2019
+description: Create dialog boxes through custom functions in Excel using JavaScript.
+title: Custom functions dialogs (preview)
+localization_priority: Priority
+---
+# Display a dialog box in custom functions
+
+Within your add-in, you have two options for creating dialogs:
+
+- `OfficeRuntime.Dialog` can be accessed by both the browser-based JavaScript engine and custom functions runtime. You should use this dialog when developing custom functions.
+- The [Dialog API](https://docs.microsoft.com/office/dev/add-ins/develop/dialog-api-in-office-add-ins) is used by Office add-in's browser-based Javascript engine, but is not accessible from custom functions' runtime.
```

Comment:
for links on docs, you can use relative link i.e., remove https://docs.microsoft.com

Reply:
Right oh! Sorry about that